ansible/roles/vendor/requirements.yml

```diff
@@ -59,7 +59,7 @@
   version: v2.2.3
 - name: gsa.datagov-deploy-jumpbox
   src: https://github.com/GSA/datagov-deploy-jumpbox
-  version: v1.1.1
+  version: v2.0.0
```

Comment:
Just curious - where does ansible versioning happen? I thought it might be in gsa.datagov-deploy-jumpbox/meta/main.yml, but seems not.

Reply:
It's by GitHub tag, unless published to https://galaxy.ansible.com (which we don't do).

Ansible role versioning is not great, but usable.